src/math/warpExposure.cc

```diff
@@ -535,7 +535,8 @@ int warpImage(DestImageT &destImage, SrcImageT const &srcImage,
     std::vector<double> const srcParentToLocalVec = {static_cast<double>(-srcImage.getX0()),
                                                      static_cast<double>(-srcImage.getY0())};
     auto const srcParentToLocalMap = ast::ShiftMap(srcParentToLocalVec);
-    auto const localDestToSrcMap = srcParentToLocalMap.of(destToSrc.getFrameSet()->of(destLocalToParentMap));
+    auto const localDestToSrcMap =
+            destLocalToParentMap.then(*(destToSrc.getFrameSet())).then(srcParentToLocalMap);
```

Comment:
If you mean "why not use destToSrc directly instead of getting the FrameSet from it?"....that's because everything else is a Mapping and I don't want to mix Mapping and Transform in my concatenation. And I don't want to use Transforms when Mappings will do just fine. I suppose I could change that.